CHANGELOG.md

```diff
@@ -46,7 +46,7 @@ _None._
 
 ### Internal Changes
 
-_None._
+- Update the CI Xcode image to `14.3.1` and CI Toolkit to `2.17.0` [#258]
```

Comment:
FWIW, I don't see the need to track this information in the CHANGELOG. Granted, it is an internal change, but it does not affect the library's behavior nor requires developers to be mindful something different in the code.

Reply:
Thanks for the feedback! I went ahead and removed it.